src/Layers/xrRender/blenders/Blender_Recorder.cpp

```diff
@@ -357,3 +360,14 @@ void CBlender_Compile::Stage_Constant(LPCSTR name)
     int id = ParseName(name);
     passConstants.push_back(RImplementation.Resources->_CreateConstant((id >= 0) ? *lst[id] : name));
 }
+
+pcstr CBlender_Compile::getSubPath(pcstr spt)
+{
+    string_path tmp, pth;
+    strconcat(sizeof(pth), pth, GEnv.Render->getShaderPath(), spt);
+    FS.update_path(tmp, "$game_shaders$", pth);
+    if (FS.folder_exist(tmp))
```

Comment:
Why you want to check availability of the feature checking the existance of the folder? Maybe it will be better to use configs?